### DISCUSSION TITLE
Media Query Initial Load on Smaller Screen

### DISCUSSION THREAD

**Discussion by matt1208** [chakra-ui/chakra-ui#3741] [Q&A]
I am using a Media Query to change the layout of a card to be more responsive, so I have built two some what different cards and placed them in a media query to change. On the initial load of the page for a bigger screen it is totally find and if you shrink the window size the responsiveness acts perfect. However, when you do an initial load of the page on a smaller screen it doesn't load the correct smaller screen version of the card. I have hard this problem multiple times now when using Media Queries, and ideas on how to fix it?
This picture is the card on initial load on larger screen: 
![image](https://user-images.githubusercontent.com/46036105/113605504-9b506800-95fb-11eb-82d2-6efecc33c652.png)

This picture is the card on initial load on smaller screen, with the problem :
![image](https://user-images.githubusercontent.com/46036105/113605631-c8047f80-95fb-11eb-89af-22f9aa520003.png)

How it should look on smaller screen: 
![image](https://user-images.githubusercontent.com/46036105/113605712-de124000-95fb-11eb-939b-1a782d01dac1.png)
 
 Code Below
 ` 
{isLargerThan1100 ? (
                <>
                    <Box
                        width="100%"
                        backgroundColor={boxbg}
                        borderRadius="10px"
                        p={4}
                        height="250px"
                    >

                        <Box>
                            <Center><Image src="/Group 11.svg" width="250px"></Image></Center>
                        </Box>

                        <Flex
                            flexDirection="column"
                            // marginBottom="20px"
                            pl="2"
                        >
                            <Text color={color} fontWeight="bold">Test</Text>
                            <Heading size="lg" color={color}>Test</Heading>
                        </Flex>
                    </Box>
                </>
            ) : (
                <Box
                    width="100%"
                    backgroundColor={boxbg}
                    borderRadius="10px"
                    p={4}
                    height="250px"
                >

                    <SimpleGrid columns={2}>
                        <Flex
                            flexDirection="column"
                            // marginBottom="20px"
                            marginTop="160px"
                            pl="2"
                        >
                            <Text color={color} fontWeight="bold">Test</Text>
                            <Heading size="lg" color={color}>Test</Heading>
                        </Flex>

                        <Box marginLeft="30%">
                            <Image src="/Group 11.svg" width="400px"></Image>
                        </Box>
                    </SimpleGrid>
                </Box>
            )
            }
`

**Comment by malashevskyi** [comment]
Try to reproduce your problem on [codesandbox](https://codesandbox.io/s/gallant-sunset-nybps)
